app/modules/proposal/views/proposal.html

```diff
@@ -0,0 +1,66 @@
+<div class="content">
+                <div class="page-title">
+                    <h3>Proposal</h3>
+                </div>
+                <div id="userList"  ng-init="userList()">
+                
+                    <div class="row">
+                    <div class="col-md-12">
+                        <div class="grid simple">
+                        <div class="grid-title no-border">
+                            <h4 class="tip" data-toggle="tooltip" title="PSJA Users"><span class="semi-bold"> </span>Proposal List</h4>
+                            <div class="tools">
+                                <a href="javascript:;" class="reload"></a>
+                            </div>
+                            <input type="text" ng-model="searchname">
+                            </div>
+
+                        <div class="grid-body no-border">
+                            <div class="row">
+                            <table class="table no-more-tables">
+                                <thead>
+                                    <tr>
+                                       
+                                        <th style="width:22%">Proposal Name</th>
+                                        <th style="width:16%">Assigned To</th>
+                                        <th style="width:28%">Client Name</th>
+                                        <th style="width:5%"></th>
+                                    </tr>
+                                </thead>
+                                <tbody>
+                                    <tr ng-repeat="singleUser in getProposals">
```

Comment:
@gauravc-cuelogic @sandeepp-cuelogic there should be provision to show message if there is no proposal.